lib/python/qmk/questions.py

```diff
@@ -0,0 +1,103 @@
+"""Functions to collect user input.
+"""
+
+from milc import cli, format_ansi
+
+
+def type_unchanged(value):
+    """Return the object without changing its type.
+    """
+    return value
```

Comment:
Is this function necessary? I might just not see its reason though...

Reply:
It's a way to keep the code simpler down below. Looking at this again with fresh eyes I can probably replace it with `str()` instead.

Reply:
Great! It's cleaner and easier to read this way.